bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_generic.py

```diff
@@ -28,3 +28,7 @@ def pythonizegeneric(klass, name):
     AddPrettyPrintingPyz(klass)
 
     return True
+
+# Add free functions to the ROOT module
+import cppyy
+cppyy.gbl.AsRVec = AsRVec
```

Comment:
I am not sure about this. Two things make me feel uneasy:
- `AsRVec` being injected in the global namespace.
- The fact that this is injected in `_generic.py`. Generic pythonisations should be those that affect all classes, this is only related to `RVec`, even if it is not a pyhonisation of the `RVec` class per se.

Reply:
I agree. Do you have an idea where to put it?

Reply:
But this is kind of ugly?

Reply:
On the one hand it is uglier, but on the other it makes the function be better classified. What do others think @dpiparo ?

Reply:
And I would put it in `_rvec.py` since I do not consider it generic.

Reply:
+1 to Enric's comments here.

Reply:
I support the adoption of the conventions of file naming.
For the qualification of the utility, the VecOps namespace can be probably left aside since in C++ RVec is reachable also from the ROOT namespace. For this particular case the usage of the global namespace would not be the end of the world, but in the medium term we risk to clutter the bindings if we do not impose to ourselves strict rules..

Reply:
I fully agree regarding the naming of the files. however, in which scope should the user find the utility? `ROOT.AsRVec`? `ROOT.ROOT.VecOps.AsRVec`?

Either we put it into the global namespace of the python module and get `ROOT.AsRVec` or we put it in a C++ namespace and get a `ROOT.ROOT.<namespace>.AsRVec` as syntax.

Reply:
Let's see, I believe the votes so far are:
- `ROOT.AsRVec` (Stefan)
- `ROOT.ROOT.AsRVec` (Danilo)
- `ROOT.ROOT.VecOps.AsRVec` (Enric, Guilherme)
I could also do 2, but I would prefer not 1.

Reply:
For me, option 1 and 2 are the same (from the Python view). The additional `ROOT.ROOT` is just a relic of the C++ namespace.

Reply:
We agreed to put it into the `ROOT.ROOT.VecOps.AsRVec` namespace since we can always promote it from there in the future.